### PR TITLE
Improvements to raw data plotter

### PR DESCRIPTION
**Pull request by SamStruthers**
Made some small changes to the tools I use: `download_pictures` and `photo_organizer. These don't affect any other workflow but should hopefully help clean up field photo stuff. 

I added some text to the 01 and 02 scripts in `virridy_verification`, mostly just to help techs/others if they run into lil errors. If you have run into other issues that would be useful to note, lets add em!

Raw data plotter has the most changes. It looks more similar to the weekly plots in the verifcation workflow with other sites as lines and the site being verified plotted as points. 
- You now select your primary site (the one you would be verifying) and it will be displayed as points
- Additional sites are plotted as lines
- There is now a button to turn flags on and off. This changes the colors of the points and allows you to toggle them on and off thru plotly
- I went ahead an removed any flag with 'missing data' since it wouldn't show up on the plot anyways.

Review Requested: Just make sure it works on your end! And/or if you hate it, we can change it back to its original form :)

### CODE REVIEW

File: virridy/verification_workflow/raw_data_plotter.R [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+        } else {
+
+          #Only log 10
+
+          p <- base_plot +
+            geom_point(data = trim_select_data %>% filter(site == input$site_oi),
+                       aes(x = DT_round, y = mean, colour = site)) +
+
+            scale_y_continuous(trans = "log",
```
Comment (kathryn-willi) [inline virridy/verification_workflow/raw_data_plotter.R:141]:
```suggestion
            scale_y_continuous(trans = "log10",
```

File: virridy/verification_workflow/raw_data_plotter.R [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        # Plot data and apply log scale to selected parameters
+        if(input$flags_TF == "yes"){
+
+          # Flags + log 10
+          p <- base_plot +
+            geom_point(data = trim_select_data %>% filter(site == input$site_oi),
+                       aes(x = DT_round, y = mean, colour = flag)) +
+
+            labs(colour = "Flags on data", color = "Additional Sites")+
+            scale_y_continuous(trans = "log",
```
Comment (kathryn-willi) [inline virridy/verification_workflow/raw_data_plotter.R:128]:
```suggestion
            scale_y_continuous(trans = "log10",
```

Comment (kathryn-willi) [reply virridy/verification_workflow/raw_data_plotter.R:128]: This is a screw up I did! Setting to base 10 log. Shouldn't impact things too much visually at all.

**Review by kathryn-willi — APPROVED** [review]
Looks great and ran fine on my end!!!

**Review by juandlt-csu — APPROVED** [review]
Thanks for making these updates, Sam! Looks good on my end.

**Pull request merged**